packages/terra-consumer-nav/src/components/user-profile/UserProfile.scss

```diff
@@ -9,6 +9,13 @@
     position: fixed;
     width: var(--terra-consumer-profile-link-width, 300px);
 
+    @media screen and (min-width: $terra-medium-breakpoint) {
```

Comment:
since there are other places using media query for mobile with
```
max-width: $terra-medium-breakpoint
```
so it should be $terra-medium-breakpoint +1 to avoid the inconsistency at that breakpoint